tests/Integration/Behaviour/Features/Context/Domain/Discount/DiscountFeatureContext.php

```diff
@@ -208,6 +249,16 @@ protected function createDiscount(string $cartRuleReference, array $data, AddDis
             }
         }
 
+        if ($command instanceof AddProductLevelDiscountCommand) {
+            if (!empty($data['reduction_product'])) {
+                if ((int) $data['reduction_product'] === -1 || (int) $data['reduction_product'] === -2) {
```

Comment:
Similar here when we handle these values properly we could use human-readable string values to identify them, but it's fine for now